src/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.jsx

```diff
@@ -89,27 +91,26 @@ const DotPlotPage = (props) => {
     error: plotDataError,
   } = useSelector((state) => state.componentConfig[plotUuid]) || {};
 
-  const {
-    fetching: genesFetching,
-    data: highestDispersionGenes,
-  } = useSelector((state) => state.genes.properties.views[plotUuid] || {});
+  const { data: geneData } = useSelector((state) => state.genes.properties || {});
 
   const cellSets = useSelector(getCellSets());
 
   const {
     loading: cellSetsLoading,
     error: cellSetsError,
-    hierarchy: cellSetHierarcy,
+    hierarchy: cellSetHierarchy,
     properties: cellSetProperties,
   } = cellSets;
 
   const [moreThanTwoGroups, setMoreThanTwoGroups] = useState(false);
+  const [reorderFetched, setReorderFetched] = useState(false);
```

Comment:
what is `reorderFetched`? Is it whether new genes have been fetched after reordering? Can you rename this variable to make it easier to understand?

Reply:
I renamed it to `reorderAfterFetch`. It's used to enable a `useEffect` where adding a gene to the plot (via search box) will trigger the data to be reordered according to the order of selected genes - it's needed because `fetchWork` "forgets" the previous order of genes.